Source/Core/Core/DSP/Interpreter/DSPIntTables.cpp

```diff
@@ -0,0 +1,391 @@
+  {0x3000, 0xfc80, xorr},
+  {0x3400, 0xfc80, andr},
+  {0x3800, 0xfc80, orr},
+  {0x3c00, 0xfe80, andc},
+  {0x3e00, 0xfe80, orc},
+  {0x3080, 0xfe80, xorc},
+  {0x3280, 0xfe80, notc},
+  {0x3480, 0xfc80, lsrnrx},
+  {0x3880, 0xfc80, asrnrx},
+  {0x3c80, 0xfe80, lsrnr},
+  {0x3e80, 0xfe80, asrnr},
+
+  // 4
+  {0x4000, 0xf800, addr},
+  {0x4800, 0xfc00, addax},
+  {0x4c00, 0xfe00, add},
+  {0x4e00, 0xfe00, addp},
+
+  // 5
+  {0x5000, 0xf800, subr},
+  {0x5800, 0xfc00, subax},
+  {0x5c00, 0xfe00, sub},
+  {0x5e00, 0xfe00, subp},
+
+  // 6
+  {0x6000, 0xf800, movr},
+  {0x6800, 0xfc00, movax},
+  {0x6c00, 0xfe00, mov},
+  {0x6e00, 0xfe00, movp},
+
+  // 7
+  {0x7000, 0xfc00, addaxl},
+  {0x7400, 0xfe00, incm},
+  {0x7600, 0xfe00, inc},
+  {0x7800, 0xfe00, decm},
+  {0x7a00, 0xfe00, dec},
+  {0x7c00, 0xfe00, neg},
+  {0x7e00, 0xfe00, movnp},
+
+  // 8
+  {0x8000, 0xf700, nx},
+  {0x8100, 0xf700, clr},
+  {0x8200, 0xff00, cmp},
+  {0x8300, 0xff00, mulaxh},
+  {0x8400, 0xff00, clrp},
+  {0x8500, 0xff00, tstprod},
+  {0x8600, 0xfe00, tstaxh},
+  {0x8a00, 0xff00, srbith},
+  {0x8b00, 0xff00, srbith},
+  {0x8c00, 0xff00, srbith},
+  {0x8d00, 0xff00, srbith},
+  {0x8e00, 0xff00, srbith},
+  {0x8f00, 0xff00, srbith},
+
+  // 9
+  {0x9000, 0xf700, mul},
+  {0x9100, 0xf700, asr16},
+  {0x9200, 0xf600, mulmvz},
+  {0x9400, 0xf600, mulac},
+  {0x9600, 0xf600, mulmv},
+
+  // A-B
+  {0xa000, 0xe700, mulx},
+  {0xa100, 0xf700, abs},
+  {0xa200, 0xe600, mulxmvz},
+  {0xa400, 0xe600, mulxac},
+  {0xa600, 0xe600, mulxmv},
+  {0xb100, 0xf700, tst},
+
+  // C-D
+  {0xc000, 0xe700, mulc},
+  {0xc100, 0xe700, cmpar},
+  {0xc200, 0xe600, mulcmvz},
+  {0xc400, 0xe600, mulcac},
+  {0xc600, 0xe600, mulcmv},
+
+  // E
+  {0xe000, 0xfc00, maddx},
+  {0xe400, 0xfc00, msubx},
+  {0xe800, 0xfc00, maddc},
+  {0xec00, 0xfc00, msubc},
+
+  // F
+  {0xf000, 0xfe00, lsl16},
+  {0xf200, 0xfe00, madd},
+  {0xf400, 0xfe00, lsr16},
+  {0xf600, 0xfe00, msub},
+  {0xf800, 0xfc00, addpaxz},
+  {0xfc00, 0xfe00, clrl},
+  {0xfe00, 0xfe00, movpz},
+}};
+
+constexpr std::array<InterpreterOpInfo, 25> s_opcodes_ext
+{{
+  {0x0000, 0x00fc, Ext::nop},
+
+  {0x0004, 0x00fc, Ext::dr},
+  {0x0008, 0x00fc, Ext::ir},
+  {0x000c, 0x00fc, Ext::nr},
+  {0x0010, 0x00f0, Ext::mv},
+
+  {0x0020, 0x00e4, Ext::s},
+  {0x0024, 0x00e4, Ext::sn},
+
+  {0x0040, 0x00c4, Ext::l},
+  {0x0044, 0x00c4, Ext::ln},
+
+  {0x0080, 0x00ce, Ext::ls},
+  {0x0082, 0x00ce, Ext::sl},
+  {0x0084, 0x00ce, Ext::lsn},
+  {0x0086, 0x00ce, Ext::sln},
+  {0x0088, 0x00ce, Ext::lsm},
+  {0x008a, 0x00ce, Ext::slm},
+  {0x008c, 0x00ce, Ext::lsnm},
+  {0x008e, 0x00ce, Ext::slnm},
+
+  {0x00c3, 0x00cf, Ext::ldax},
+  {0x00c7, 0x00cf, Ext::ldaxn},
+  {0x00cb, 0x00cf, Ext::ldaxm},
+  {0x00cf, 0x00cf, Ext::ldaxnm},
+
+  {0x00c0, 0x00cc, Ext::ld},
+  {0x00c4, 0x00cc, Ext::ldn},
+  {0x00c8, 0x00cc, Ext::ldm},
+  {0x00cc, 0x00cc, Ext::ldnm},
+}};
+// clang-format on
+
+namespace
+{
+std::array<InterpreterFunction, 65536> s_op_table;
```

Comment:
That number freaked me out a little, but then I noticed how it was already part of the original tables. I guess we can spare the extra megs there...